lib/inferno/dsl/fhir_evaluator/evaluator_util.rb

```diff
@@ -0,0 +1,127 @@
+module Util
+  # General util methods
+  def self.get_meta_profile(resource)
+    resource&.meta&.profile || []
+  end
+
+  # reference resolution methods
+  def self.extract_ids_references(resources)
+    extractor = ReferenceExtractor.new
+    extractor.extract_ids_references(resources)
+  end
+
+  def self.validate_resource(resource)
```

Comment:
This is going to be replaced by the existing validator classes, so this method can be removed

Reply:
Agreed. Removed.